app/controllers/api/v2/compute_resources_controller.rb

```diff
@@ -46,7 +46,7 @@ def show
           param :server, String, :desc => N_("for VMware")
           param :set_console_password, :bool, :desc => N_("for Libvirt and VMware only")
           param :display_type, %w(VNC SPICE), :desc => N_('for Libvirt and oVirt only')
-          param :keyboard, ComputeResource::ALLOWED_KEYBOARD_LAYOUTS, :desc => N_('for oVirt only')
+          param :keyboard_layout, ComputeResource::ALLOWED_KEYBOARD_LAYOUTS, :desc => N_('for oVirt only')
```

Comment:
Should we move the `ALLOWED_KEYBOARD_LAYOUTS` to `Foreman::Model::Ovirt` in a follow-up?